src/Sets/Tetrahedron.jl

```diff
@@ -0,0 +1,111 @@
+export Tetrahedron
+
+"""
+    Tetrahedron{N, VN<:AbstractVector{N}, VT<:AbstractVector{VN}} <: AbstractPolytope{N}
+
+Type that represents a tetrahedron in vertex representation.
+
+### Fields
+
+- `vertices` -- list of vertices
+
+### Examples
+
+A tetrahedron can be constructed by passing the list of vertices.
+The following builds the tetrahedron with edge length 2 from the [wikipedia page Tetrahedron](https://en.wikipedia.org/wiki/Tetrahedron): 
+
+```jldoctest
+julia> vertices = [[1, 0, -1/sqrt(2)], [-1, 0, -1/sqrt(2)], [0, 1, 1/sqrt(2)], [0, -1, 1/sqrt(2)]];
+
+julia> T = Tetrahedron(vertices); T isa Tetrahedron
+true
```

Comment:
I find writing two commands in one line hard to read, and `T isa Tetrahedron` is also not very surprising.

Reply:
ok ill remove the `isa`